src/components/SellItemForm.tsx

```
@@ -44,6 +45,17 @@ export function SellItemForm({ onClose, onSuccess }: SellItemFormProps) {
     setError(null);
 
     try {
+      // Validate stock and price
+      const stockNum = parseInt(stock);
+      if (isNaN(stockNum) || stockNum < 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)